salt/modules/boto_dynamodb.py

```diff
@@ -233,7 +265,7 @@ def describe(table_name, region=None, key=None, keyid=None, profile=None):
     return table.describe()
 
 
-def _extract_index(index_data, global_index=False):
+def extract_index(index_data, global_index=False):
```

Comment:
@patrickvalenzuela If this function doesn't make sense to call from the CLI, but you need it in other files, perhaps it makes sense to move that to the `salt.utils.boto.py` file instead?